client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -170,7 +170,7 @@ export function getEnhancedTasks(
 							window.location.assign( plansUrl );
 						},
 						badge_text: ! task.completed ? null : translatedPlanName,
-						disabled: task.completed || ! domainUpsellCompleted,
+						disabled: ( task.completed || ! domainUpsellCompleted ) && ! isBlogOnboardingFlow( flow ),
```

Comment:
Minor: My linter suggested it be on a new line:

![image](https://github.com/Automattic/wp-calypso/assets/1044309/39d6777c-c125-47ca-9fa4-be28fda4256f)

![image](https://github.com/Automattic/wp-calypso/assets/1044309/8ec33d16-2998-4212-8871-dce7192cecf0)

Reply:
This might be due to the github commit, will fix.

Reply:
oops nevermind looks like @lupus2k got to it!